libs/core/src/divider/divider.element.styles.ts

```diff
@@ -0,0 +1,14 @@
+import { css } from 'lit';
+
+export default css`
+  hr {
+    margin-block: var(--kirby-spacing-xxxs);
+    border: none;
+    border-top: 1px solid var(--kirby-divider-color);
+  }
+  :host([hasMargin='false']) hr,
+  :host([hasMargin='false']) hr[role='presentation'] {
+    margin-top: 0;
+    margin-bottom: 0;
+  }
```

Comment:
I think you need to add flip this logic - the attribute is not set on the element when false.

```suggestion
  hr {
    margin-block: 0;
    border: none;
    border-top: 1px solid var(--kirby-divider-color);
  }
  
  :host([hasMargin) hr,
    margin-block: var(--kirby-spacing-xxxs)
  }
```